index.d.ts

```diff
@@ -0,0 +1,54 @@
+type Replacer = (key: string, value: any) => void;
+
+interface Options {
+	/**
+	 * Indentation as a string or number of spaces. Pass in null for no formatting.
+	 */
+	indent?: string | number | null;
+	/**
+	 * Detect indentation automatically if the file exists.
+	 */
+	detectIndent?: boolean;
+	/**
+	 * Sort the keys recursively. Optionally pass in a compare function.
+	 */
+	sortKeys?: boolean | ((a: string, b: string) => number);
+	/**
+	 * Passed into JSON.stringify.
+	 */
+	replacer?: Replacer | Array<number | string>;
+	/**
+	 * Mode used when writing the file.
+	 */
+	mode?: number;
+}
+
+/**
+ * Stringify and write JSON to a file atomically
+ *
+ * Creates directories for you as needed.
+ *
+ * @returns {void}
+ * @example
+ * import * as writeJsonFile from 'write-json-file';
```

Comment:
Should be `import writeJsonFile from 'write-json-file';`

Reply:
🤔 ... to access `.sync()` you need to add `import * as`

Reply:
Ah, good point 👍

Reply:
I guess if you use Babel this example is correct, I'll change it to:

```js
import writeJsonFile from 'write-json-file';
```

Reply:
Most Typescript users kinda got used to adding `import * as`, no need to include it.

Most docs are mixed between Babel & Typescript, we can keep as the Babel way